studio/pages/api/projects/[ref]/analytics/endpoints/[name].ts

```diff
@@ -15,6 +15,11 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           .status(400)
           .json({ error: { message: '[analytics] is not enabled in supabase/config.toml' } })
       }
+      if (process.env.NEXT_ANALYTICS_BACKEND_PROVIDER !== 'big_query') {
+        return res
+          .status(400)
+          .json({ error: { message: 'Log Explorer is only supported by the [analytics] Big Query backend, please refer to the documentation to set it up' } })
```

Comment:
lets return a link to the user to the self-hosting analytics documentation and/or local dev logging documentation